### ISSUE TITLE
Bower blocks build on first run

### ISSUE THREAD

**Issue by victorhurdugaci** [aspnet/Mvc#2116]
On a clean machine on which Bower never ran if you build MVC you will get the following message waiting for a answer. This blocks the build script on clean build agents.

```
info: Exec
info:   program: node
info:   commandline: D:\aspnet\universe\Mvc\bin\nodejs\node_modules\bower\bin\bower install
info:   workingdir: D:\aspnet\universe\Mvc\test\WebSites\ValidationWebSite
[?] May bower anonymously report usage statistics to improve the tool over time?
```

Is there a parameter or something that you can pass to bower to not require user interaction?

**Comment by dougbu** [comment]
this should be specific to clean build agents.  otherwise we'd be seeing hangs on all existing agents.  suggest running a build interactively once when creating a new agent.  I'm not aware of a suppression option unless `--quiet` has that side-effect.

**Comment by dougbu** [comment]
fix will be in the *.shade files from the aspnet/Universe repo. tracking here because symptoms could be visible in our builds.

**Comment by dougbu** [comment]
fixed in commit aspnet/Universe@f6ae75e
